src/app/compare/compare-data.ts

```diff
@@ -0,0 +1,9 @@
+import { NodeLinkStrategy } from '../shared/enums/compare-method';
+
+export class CompareData {
+  id!: string;
+  originalReport: any;
+  runResultReport: any;
```

Comment:
At some point we should really try to remove as many `any` types as possible as to have accurate and consistent types throughout the whole application

Reply:
Yeah i agree, maybe the next big pr